.github/workflows/commit_artifacts.yml

```diff
@@ -165,3 +189,30 @@ jobs:
           commit_user_name: ${{ github.actor }}
           commit_user_email: ${{ github.actor }}@users.noreply.github.com
           create_branch: true
+          file_pattern: 'compiled/facebook-www/'
```

Comment:
Should be unnecessary now right?

Reply:
yep, this is not needed anymore. Thanks.